src/ui/views/HydrogenRootView.tsx

```diff
@@ -141,17 +141,11 @@ function initHydrogen() {
     return [`${protocol}//${hostname}${port ? `:${port}` : ""}`];
   })();
 
-  const config = {
-    ...configData,
-    oidc: {
-      clientConfigs: {
-        "https://id.thirdroom.io/realms/thirdroom/": {
-          client_id: oidcClientId,
-          uris: oidcUris,
-          guestKeycloakIdpHint: "guest",
-        },
-      },
-    },
+  const config = { ...configData };
+  config.oidc.clientConfigs["https://id.thirdroom.io/realms/thirdroom/"] = {
```

Comment:
Doesn't this just wipe out the config above? Is the client config you added to config.json just for an example? Should we document this client config somewhere?

Reply:
Yes this will reset the config for `https://id.thirdroom.io/realms/thirdroom/` only so it work with PR preview deploy. But if someone want to setup it with their server or local env they can add new config in `config.json` file and it will work fine.

Reply:
> Should we document this client config somewhere?

Yeah, it's a good idea if we can document how config works.